src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -0,0 +1,218 @@
+import PushNotification from 'react-native-push-notification';
+import PushNotificationIOS from '@react-native-community/push-notification-ios';
+import messaging from '@react-native-firebase/messaging';
+import React, {useState} from 'react';
+import {Platform} from 'react-native';
+import {useNavigation} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Consider using a more specific import from `@react-navigation/native` instead of `@react-navigation/core`. This will make it clear that you are using the `useNavigation` hook from React Navigation's native package.
```diff
- import {useNavigation} from '@react-navigation/core';
+ import {useNavigation} from '@react-navigation/native';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->